pos_loyalty/views/templates.xml

```diff
@@ -3,6 +3,7 @@
     <template id="assets" inherit_id="point_of_sale.assets">
         <xpath expr="." position="inside">
             <script type="text/javascript" src="/pos_loyalty/static/src/js/pos.js"></script>
+            <script type="text/javascript" src="/pos_loyalty/static/src/js/tests.js"></script>
```

Comment:
Is it a good way to include JS test. I mean, this file will be loaded allways, in production, even if it's not necessary.

regards.

Reply:
I can't see a reliable way on how to distinguish between a production and not-production session. Please share if you have an idea.

Also the inherited template already includes the tests from the point_of_sale so I don't see why this should be considered as an exception.

Reply:
Excuse me, it was not about production or non production but about with demo data or without. In that case, adding a new xml file, imported in the 'demo' data of the manifest will let it available for test (what we want) but avoid to load it on production. (but yes, it should be done in the same inherit_id = point_off_sale.assets".
What do you think ?

Reply:
That is a good idea, it depends on demo data so it wont work if that isn't loaded anyway. Thanks, I'll make the change